pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -703,6 +703,10 @@ private static Expression compileFunctionExpression(SqlBasicCall functionNode) {
         negated = ((SqlLikeOperator) functionNode.getOperator()).isNegated();
         canonicalName = SqlKind.LIKE.name();
         break;
+      case CONTAINS:
+        negated = ((SqlLikeOperator) functionNode.getOperator()).isNegated();
```

Comment:
(MAJOR) this is not `SqlLikeOperator`, and will cause cast exception. No need to add `CONTAINS` branch into this switch, the default handling should be good for it

Reply:
Ack, removed